calculator/operations.py

```diff
@@ -1,23 +1,43 @@
+from __future__ import division
 
 def add(*args):
-    # your implementation here
-    pass
+        if len(args) == 1:
+            print(args[0])
+            return args[0]
+        print (sum(args))
+        return sum(args)
 
 def subtract(*args):
-    # your implementation here
-    pass
-
+        if len(args) == 1:
+            print(args[0])
+        print (args[0] - sum(args[1::]))
+        return args[0] - sum(args[1::])
+    
 def multiply(*args):
-    # your implementation here
-    pass
-
+    multiplication = 1
+    for item in args:
+        multiplication = multiplication * item
+        print (multiplication)
+    return multiplication
+        
 def divide(*args):
-    # your implementation here
-    pass
+    if len(args)==1:
+        return args[0]
+    division=args[0]    
+    for item in args[1::]:
+        division=division/item
+    print (division)
+    return division
 
```

Comment:
I suggest that the print statements be removed or commented out once the code is verified... good for initial troubleshooting but would generate lot of output